### ISSUE TITLE
does not work well when text is pre-set

### ISSUE THREAD

**Issue by vonovak** [zbtang/React-Native-TextInputLayout#3]
when the text is pre-set, the floating label stays down. 
also consider the case when new text is simply passed as new prop

this might fix the first issue it but I cannot test it very well atm

```
    _handleChildren(props) {
        let edtChild = React.Children.only(props.children);
        this._oriEdtChild = edtChild;
        this._oriEdtStyle = StyleSheet.flatten([edtChild.props.style])
        this._oriOnFocus = edtChild.props.onFocus;
        this._oriOnBlur = edtChild.props.onBlur;
        this._oriOnChangeText = edtChild.props.onChangeText;
+        const textValue = edtChild.props.value
+        if (textValue) {
+            this._edtText = textValue
+            this.state.showLabel = true
+            this.state.labelAnimationValue = new Animated.Value(1)
+        }
....
```

**Comment by nupejosh** [comment]
@vonovak your fix worked for me.. maybe submit a PR

**Comment by waqas19921** [comment]
Having same problem when text is preset placeholder comes over the text.

**Comment by waqas19921** [comment]
@vonovak your fix worked for me also. Please submit a PR...

**Comment by zbtang** [comment]
Sorry for no reply for a long time, a little busy these days. This bug has been fix on version 1.0.1, you can test it. Thanks !
